kube/services/monitoring/prometheus-values.yaml

```diff
@@ -784,7 +784,7 @@ server:
 
   ## Prometheus data retention period (i.e 360h)
   ##
-  retention: "360h"
+  retention: "240h"
```

Comment:
So we will only keep metrics for 10 days, instead of 15 days?
Do we have any growth trends reports on this disk space?